### ISSUE TITLE
No theme when built on Debian 9.8

### ISSUE THREAD

**Issue by mrkafk** [mank319/Go-For-It#139]
### Expected behavior

Theme works like in flatpak version.

### Actual behavior

Controls are small and it seems like theme is not used. See attached screenshot.

When switching theme in settings following messages are displayed in console:

```
(com.github.jmoerman.go-for-it:65762): Gtk-WARNING **: Theme parsing error: gtk.css:68:35: The style property GtkButton:child-displacement-x is deprecated and shouldn't be used anymore. It will be removed in a future version

(com.github.jmoerman.go-for-it:65762): Gtk-WARNING **: Theme parsing error: gtk.css:69:35: The style property GtkButton:child-displacement-y is deprecated and shouldn't be used anymore. It will be removed in a future version

(com.github.jmoerman.go-for-it:65762): Gtk-WARNING **: Theme parsing error: gtk.css:70:34: The style property GtkCheckButton:indicator-size is deprecated and shouldn't be used anymore. It will be removed in a future version

(com.github.jmoerman.go-for-it:65762): Gtk-WARNING **: Theme parsing error: gtk.css:71:36: The style property GtkCheckMenuItem:indicator-size is deprecated and shouldn't be used anymore. It will be removed in a future version

(com.github.jmoerman.go-for-it:65762): Gtk-WARNING **: Theme parsing error: gtk.css:73:46: The style property GtkScrolledWindow:scrollbars-within-bevel is deprecated and shouldn't be used anymore. It will be removed in a future version
```
![gfi_theme_missing](https://user-images.githubusercontent.com/593219/53649874-80081700-3c43-11e9-89de-d4fb840d95fc.png)

### How to reproduce the behavior

Build on Debian 9.8.

Used cmake cmd:

```
cmake -DCMAKE_INSTALL_PREFIX=/usr/local/gfi
```

Running:
```
cd /usr/local/gfi/bin

./com.github.jmoerman.go-for-it 
```

### Operating System used

Debian 9.8  amd64 running in Virtualbox.

Desktop: **KDE Plasma**.

### Installation source

Cloned github source.

**Comment by JMoerman** [comment]
_Go For It!_ 1.6.x uses `XDG_DATA_DIRS` to find its css file and you haven't added `/usr/local/gfi` to this environment variable.
This behaviour is a relic from a past where the build system wasn't used to generate Constants.vala so the binary would be the same for each installation location.
This is now fixed in the development branch https://github.com/JMoerman/Go-For-It

**Comment by JMoerman** [comment]
I've back ported the fix to 1.6 ([1.6.11](https://github.com/mank319/Go-For-It/releases/tag/1.6.11)).

**Comment by JMoerman** [comment]
(The errors you reported seem to be from breeze-gtk. Gtk.css isn't a file that is loaded by _Go For It!_.)
